pyproject.toml

```
@@ -1,2 +1,10 @@
 [build-system]
-requires = ["setuptools >= 40.8.0", "wheel"]
+requires = ["setuptools ~= 42.0", "wheel", "setuptools_scm[toml] >= 3.4"]
```

Elsewhere it looks like you're permitting setuptools 44. I'm guessing this is a typo.

```suggestion
requires = ["setuptools >= 42.0", "wheel", "setuptools_scm[toml] >= 3.4"]
```